### ISSUE TITLE
Unnecessary count operations in Data Browser

### ISSUE THREAD

**Issue by dplewis** [parse-community/parse-dashboard#2246]
### New Issue Checklist

- [x] I am not disclosing a [vulnerability](https://github.com/parse-community/parse-server/blob/master/SECURITY.md).
- [x] I am not just asking a [question](https://github.com/parse-community/.github/blob/master/SUPPORT.md).
- [x] I have searched through [existing issues](https://github.com/parse-community/parse-dashboard/issues?q=is%3Aissue).
- [x] I can reproduce the issue with the latest versions of [Parse Server](https://github.com/parse-community/parse-server/releases) and [Parse Dashboard](https://github.com/parse-community/parse-dashboard/releases). 

### Issue Description

Count operations are very expensive and are run every time the data browser changes classes.

https://github.com/parse-community/parse-dashboard/blob/alpha/src/dashboard/Data/Browser/Browser.react.js#L176

### Steps to reproduce

Inspect element on your browser and load the page. The count operations run for every class. Switching classes will also run count operations (easier to notice on classes with a lot of objects). 

### Actual Outcome

Count runs once the page loads and every change between classes.

### Expected Outcome

Run count only once when the page loads

### Environment

Dashboard
- Parse Dashboard version: `4.1.4`
- Browser (Safari, Chrome, Firefox, Edge, etc.): `Chrome`
- Browser version: `103.0.5060.134 `

Server
- Parse Server version: `4.1.2`
- Operating system: `Mac`
- Local or remote host (AWS, Azure, Google Cloud, Heroku, Digital Ocean, etc): `local`

Database
- System (MongoDB or Postgres): `Mongo`
- Database version: `4.2`
- Local or remote host (MongoDB Atlas, mLab, AWS, Azure, Google Cloud, etc): `Atlas`

**Comment by mtrezza** [comment]
Closing via https://github.com/parse-community/parse-dashboard/pull/2250
